ci/test/slt-fast.sh

```diff
@@ -159,4 +159,10 @@ tests=(
     test/sqllogictest/postgres/pgcrypto/*.slt
 )
 
+tests_with_views=(
+    test/sqllogictest/aggregates.slt
```

Comment:
Does it make sense to still run aggregates.slt without `--auto-index-selects` in addition?

Reply:
Another thing is, maybe we should turn around the logic here and have `tests_without_views`, if it's not too many, so that new tests are run with --auto-index-selects by default.

Reply:
This is a great suggestion. I have added a commit now that runs the fast SQL logic tests with `--auto-index-selects` by default, following a similar structure to your commit for the slow SQL logic tests.